### DISCUSSION TITLE
kids own dashboard

### DISCUSSION THREAD

**Discussion by robnieuw** [home-assistant/frontend#16590] [Accessibility]
Hello,

I would like to be able to give my children a profile with which they can manage certain items. Like the lamps in their bedroom. But that they cannot change the temperature in the house. So profile "kids" can only control item X,Y and Z.
one dashboard per user.

regards robert

**Comment by Sprawlie** [comment]
Create a user per kid. Lock them exclusively to their own dashboards  (you will have to create them)

**Reply by BoneheadFraggle** [reply]
..until they find the letter e on the keyboard, and suddenly they explore all unexpected possibilities in HA :-)

My point: HA needs better possibilities to lock down features. For kids, guests and so on.
There should be an "everything is locked down by default, allow the features you want for this specific user". Now it's the other way around - everything is possible and you need to use ugly manual hacks to lock down.

**Reply by Xitee1** [reply]
That would definitely be the better solution. A good permission management system where you can define which user should be able to see which dashboards, which entities/devices are accessible, what he is allowed to do or see in the settings (e.g. creating scenes but no automations/scripts,..), is he allowed to modify (own) dashboards and so on.

**Comment by dhoeben** [comment]
I up this one! I just made a feature request for an ‘child’ modus in the person category.
